### PR TITLE
Add other talks

### PR DESCRIPTION
**Pull request by iusildra**

### CODE REVIEW

**Review by markarasev — APPROVED** [review]

File: src/main/scala/io/scala/modules/TalkCard.scala [hunk 1]
```diff
@@ -36,38 +37,41 @@ object TalkCard:
...
             )
-          )
-        },
-        button(
-          className  := "link classy-button highlight",
-          aria.label := s"Go to ${talk.info.title}",
+          }
+        ),
+        Buttons.classyNew(
+          true,
```
[the remark this answers is not in the document]
Comment (iusildra) [reply src/main/scala/io/scala/modules/TalkCard.scala:71]: Yeah, I'm trying to adopt a more "design system" approach so this'll probably change later 
Maybe with some extensions to have a builder-like pattern 🤷

**Comment by iusildra** [comment]
There are errors in the speakers' data, at least for Tomas Mikula, I'll need to fix it before merging (unless you can double check and fix issues now ^^)

File: src/main/scala/io/scala/modules/TalkCard.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (markarasev) [reply src/main/scala/io/scala/modules/TalkCard.scala:71]: You know this codebase way better than me but my usual advice when I hear "builder pattern" is : keep it simple and stupid 😉

[the remark this answers is not in the document]
Comment (iusildra) [reply src/main/scala/io/scala/modules/TalkCard.scala:71]: I thought of reusing current element definition and my "builder" idea was to juste append the css class
```scala
extension (e: ...)
  def important = e.append(className := "important")
```
It's not a real builder, but that's the spirit though

[the remark this answers is not in the document]
Comment (markarasev) [reply src/main/scala/io/scala/modules/TalkCard.scala:71]: looks nice!

**Pull request merged**